### ISSUE TITLE
Revisit omx input files setup

### ISSUE THREAD

**Issue by nkuehnel** [msmobility/mito#55]
OMX allows to store multiple matrices in one file. We could use that to save multiple skims (e.g. car, pt, walk etc) in one file, effectively reducing the amount of input files and properties needed. We can still address different matrices in the same file.

**Comment by moeckel** [comment]
Absolutely, yes. We should store multiple matrices in one file (as long as the dimensions of the matrices are equal and they fit thematically into one file).

**Comment by nkuehnel** [comment]
goes hand in had with 
msmobility/silo#218
